packages/core/src/tools.ts

```diff
@@ -72,8 +76,13 @@ export namespace Tools {
 		providedOptions: any
 	) {
 		defaultOptions = Tools.clone(defaultOptions);
+		const providedChartTypes = providedOptions.chartTypes;
 		const providedAxesNames = Object.keys(providedOptions.axes || {});
 
+		if (providedChartTypes) {
+			defaultOptions.chartTypes = providedChartTypes;
+		}
```

Comment:
why do we need this here? wouldn't these move over with the `Tools.merge()` call?

Reply:
there was some weird things happening here under the hood, i changed it quite a bit (here and in 2 other places)